data/human/free worlds start.txt

```diff
@@ -616,10 +616,18 @@ mission "FW Hope Recon 1C"
 			`	The landing crew has absolutely nowhere to hide in this glacial field; you pick them off easily. There is no response from the ship, so you land beside it and prepare to board it. But, the moment you open your hatch, the other ship roars to life, powering up its engines and beating a hasty retreat while firing a few parting shots at you from its turret. You fire up your own engines and pursue it...`
 				launch
 			
-			label spare
+			label accept
 			apply
 				karma ++
-			`	Most ships are equipped with external speakers for exactly this sort of situation. Speaking through them, you tell the Navy crew below, "Return to your ship and leave this system immediately." They beat a hasty retreat and fire up their engines. You follow their ship into orbit to make sure it truly does leave the system...`
+				set "fw accepted Susquehanna's word of honor"
+			`	Most ships are equipped with external speakers for exactly this sort of situation.`
+			`	"Crew of the R.N.S. Susquehanna, this is Captain <last> of the Free Worlds ship <ship>," you announce. "Will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`
+			`	The Gunboat captain thinks for a brief moment and gives you a thumbs-up sign, their hand held high in the air. Their crew follows suit without any visible hesitation.`
```

Comment:
It's described before that the crew raises their hands in surrender, so their hands are already held high in the air, making it weird to me to describe again that their hands our up. And as with the name of the ship, there's no information prior to this point highlighting a captain among the crew on the ground. We also know that there are still crew on the ship, and I imagine the captain would be one of those still on the ship as opposed to being on the ground.

Given that this mission makes it known that most ships have external speakers, I'd imagine Navy ships also have such speakers, and since the captain of the Gunboat is likely still on the ship, perhaps the captain uses the Gunboat's speakers to respond as opposed to relying on the hand gestures of those on the ground.

Reply:
> No, we don't know that.

We as in we the content creators, not we the players at this point in the mission.

> If the Gunboat is crewed, there's no need for speakers at all

I don't see how Navy ships being able to use radio to talk to each other precludes them from being able to have external speakers for those situations where they need to talk to someone outside the ship who doesn't have a radio. It's not as if the player's ship has speakers because it doesn't have a radio.

Reply:
> We as in we the content creators, not we the players at this point in the mission.

Even as a content creator I don't think we "know" that there's someone on board the Gunboat. We could make it so, but I don't think it's already so.

> I don't see how Navy ships being able to use radio to talk to each other precludes them from being able to have external speakers for those situations where they need to talk to someone outside the ship who doesn't have a radio. It's not as if the player's ship has speakers because it doesn't have a radio.

Sure. Of course the Gunboat _has_ speakers. It's just that if there's someone on board the Gunboat, then the Gunboat would communicate with the player by radio, not by speakers.